shuffle/cli/tests/shuffle-forge-cli-tests.rs

```diff
@@ -0,0 +1,136 @@
+    let tests = ForgeConfig::default().with_admin_tests(&[
+        &TransactionsWithoutNetworksToml,
+        &TransactionsWithoutAccount,
+        &TransactionsWithNetworkRawAddressFlags,
+    ]);
+    let options = Options::from_args();
+    forge_main(tests, LocalFactory::from_workspace()?, &options)
+}
+
+pub struct TransactionsWithoutNetworksToml;
+
+impl Test for TransactionsWithoutNetworksToml {
+    fn name(&self) -> &'static str {
+        "shuffle::transactions-without-networks-toml"
+    }
+}
+
+impl AdminTest for TransactionsWithoutNetworksToml {
+    fn run<'t>(&self, ctx: &mut AdminContext<'t>) -> Result<()> {
+        let helper = ShuffleTestHelper::new(ctx.chain_info())?;
+        let home_path_string = helper.home_path().to_string_lossy().to_string();
+        let output = std::process::Command::new(BINARY)
+            .args([
+                "--home-path",
+                home_path_string.as_str(),
+                "transactions",
+                "--network",
+                "forge",
+            ])
+            .output()?;
+        let std_err = String::from_utf8(output.stderr)?;
+        assert!(std_err
+            .contains("Error: Please add specified network to the ~/.shuffle/Networks.json\n"));
+        assert_eq!(output.status, ExitStatus::from_raw(256));
+        Ok(())
+    }
+}
+
+pub struct TransactionsWithoutAccount;
+
+impl Test for TransactionsWithoutAccount {
+    fn name(&self) -> &'static str {
+        "shuffle::transactions-without-account"
+    }
+}
+
+impl AdminTest for TransactionsWithoutAccount {
+    fn run<'t>(&self, ctx: &mut AdminContext<'t>) -> Result<()> {
+        let helper = ShuffleTestHelper::new(ctx.chain_info())?;
+        let home_path_string = helper.home_path().to_string_lossy().to_string();
+        let client = ctx.client();
+        let factory = ctx.chain_info().transaction_factory();
+        enable_open_publishing(&client, &factory, ctx.chain_info().root_account())?;
+        helper.create_project()?;
+        let output = std::process::Command::new(BINARY)
+            .args([
+                "--home-path",
+                home_path_string.as_str(),
+                "transactions",
+                "--raw",
+                "--network",
+                "forge",
+            ])
+            .output()?;
+        let std_err = String::from_utf8(output.stderr)?;
+
+        assert!(std_err
+            .contains("Error: An account hasn't been created yet! Run shuffle account first\n"));
+        assert_eq!(output.status, ExitStatus::from_raw(256));
+        Ok(())
+    }
+}
+
+pub struct TransactionsWithNetworkRawAddressFlags;
+
+impl Test for TransactionsWithNetworkRawAddressFlags {
+    fn name(&self) -> &'static str {
+        "shuffle::transactions-with-networks-raw-address-flags"
+    }
+}
+
+impl AdminTest for TransactionsWithNetworkRawAddressFlags {
+    fn run<'t>(&self, ctx: &mut AdminContext<'t>) -> Result<()> {
+        let helper = bootstrap_shuffle(ctx)?;
```

Comment:
nit: Since you selectively use this more than I expected (not used in `TransactionsWithoutAccount`), shall we rename it to 
1. `bootstrap_shuffle_project`
2. `deploy_shuffle_sample`
3. ???

Reply:
bootstrap_shuffle_project is fine. will change to this